integrations/SnapPixel/browser.js

```diff
@@ -44,6 +44,8 @@ class SnapPixel {
       AD_CLICK: "AD_CLICK",
       AD_VIEW: "AD_VIEW",
       ADD_TO_WISHLIST: "ADD_TO_WISHLIST",
+      VIEW_CONTENT: "VIEW_CONTENT",
```

Comment:
Shouldn't we remove these from `trackEvents` array?

Reply:
Done